crates/accelerate/src/equivalence.rs

```diff
@@ -340,6 +340,8 @@ impl ToPyObject for CircuitFromPython {
 type GraphType = StableDiGraph<NodeData, Option<EdgeData>>;
 type KTIType = IndexMap<Key, NodeIndex, RandomState>;
 
+/// A library providing a one-way mapping of Gates to their equivalent
```

Comment:
I think we should either reference the classes with code refs or just call them "gates" and "circuits" :)

Reply:
I'd say to go with "gates" and "circuits" because `PyO3` will reuse this docstring with Python, which may not be displayed properly.

Reply:
I think we're in agreement here, but I meant that it'd be better to not use symbol-sounding names like "QuantumCircuits" and "Gates" with a capital "G". Just call them "gates" and "circuits", verbatim.

Reply:
Also, since this is a Python class, you can use Sphinx code references if you want, instead. Like 

```
:class:`.QuantumCircuit`
```

They should turn into hyperlinks in the Python docs.

Reply:
I agree with Kevin here it should be `gates` or we do a class ref if it's a capital `G`. Although I realize the original text had `Gates` and `QuantumCircuits` without a link. We can do better here.

```suggestion
/// A library providing a one-way mapping of gates to their equivalent
```